bootstrap/gentoo.sh

```diff
@@ -1 +0,0 @@
-_gentoo_common.sh
```

Comment:
@tgurr Why did you remove the symlink? It looks like a common pattern in the codebase.

There are a lot Gentoo-based distros that this code would work for. For example, Sabayon. I just didn't have it installed to actually test the code. Also the change seem unrelated to Exherbo.